reconcile/terraform_resources.py

```diff
@@ -442,6 +442,7 @@ def run(
     # populate the resource inventory with latest output data
     populate_desired_state(ri, resource_specs)
 
+    ob.publish_metrics(ri, QONTRACT_INTEGRATION.replace("_", "-"))
```

Comment:
Why this one has a special replace, is it a requirment that `_` should be `-` in metric name? Should this be handled inside `db.publish_metrics` method?

Reply:
https://github.com/app-sre/qontract-reconcile/pull/3863/commits/7f125862723bf9e057e4b2d11cfddbc78032152f